bucket/droidcam.json

```diff
@@ -6,9 +6,6 @@
         "identifier": "Freeware",
         "url": "https://www.dev47apps.com/droidcam-terms-privacy.html"
     },
-    "suggest": {
-        "vcredist": "extras/vcredist"

```

Comment:
Its installer contains vcredist2015, so suggests `extras/vcredist2022`

Reply:
Tested it in Windows Sandbox (Windows 11) and it worked without installing vcredist, same goes for `lively`

Reply:
Required by specific functions/features, e.g. https://github.com/rocksdanister/lively/wiki/Common-Problems#web-wallpapers-not-working

I believe there are a lot of manifests that missing vcredist suggestion, Scoop doesn't handle them as a dependency, so not matter.